src/common-events/components/topProgressBar/TopProgressBar.tsx

```diff
@@ -1,8 +1,8 @@
 import { useEffect } from 'react';
 import NProgress from 'nprogress';
+import { useRouter } from 'next/router';
```

Comment:
Now, if we start using the useRouter from the next/router -package, we miss the central point were we could manage our custom router, if we want to support multiple routers. It's not decided yet what we should do with Tapahtumat-Helsinki, but it's still using a react-router instead of the next-router. Same thing with Palvelutarjotin-admin. Both the projects are or should start using the headless cms library and are some what related to events. At least the Tapahtumat-Helsinki should start using the common-events -library. The common-events library should be exported as a new plugin-module.

Reply:
I changed it so that under common-events components we use only `useRouterFromConfig` instead of using next/router straight